### PR TITLE
Updated navbar and homepage

### PR DESCRIPTION
**Pull request by Arnav717**
Added scroll down animation as well

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -28,13 +30,173 @@
...
+
+.home::before{
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background: inherit;
+      filter: blur(7px) brightness(75%);
```
Comment (darthchild) [inline css/style.css]: change the blur to 4 or 5px

File: css/style.css [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-{
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Ubuntu:wght@400;500;700&display=swap');
+
+*{
       margin: 0;
       padding: 0;
-      outline: 0;
-      border: 0;
+      box-sizing: border-box;
+      font-family: "Popins",sans-serif;
```
Comment (darthchild) [inline css/style.css:8]: try out a different font for the title text above the blurred image

**Review by darthchild — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (Arnav717) [reply css/style.css]: updated

File: css/style.css (hunk 2 above)
Comment (Arnav717) [reply css/style.css:8]: changed specifically for the title text

Comment (Arnav717) [reply css/style.css:8]: well we can change the fonts and all at the final review if something is not quite right...

**Pull request merged**